### PR TITLE
Supports Firestore task update for /api/update-task-status

### PR DESCRIPTION
**Pull request by keyonghan**
Part of https://github.com/flutter/flutter/issues/142951

### CODE REVIEW

**Review by yusuf-goog — APPROVED** [review]

File: app_dart/lib/src/request_handlers/update_task_status.dart [hunk 1]
```diff
@@ -5,14 +5,17 @@
 import 'dart:async';
 
 import 'package:gcloud/db.dart';
+import 'package:googleapis/firestore/v1.dart';
 import 'package:meta/meta.dart';
 
 import '../model/appengine/commit.dart';
 import '../model/appengine/task.dart';
+import '../model/firestore/task.dart' as f;
```
Comment (ricardoamador) [inline app_dart/lib/src/request_handlers/update_task_status.dart:13]: Should we rename this import to something like 'fire' or 'firestore' for clarity?

File: app_dart/test/request_handlers/update_task_status_test.dart [hunk 2]
```diff
@@ -4,34 +4,42 @@
 
 import 'package:cocoon_service/src/model/appengine/commit.dart';
 import 'package:cocoon_service/src/model/appengine/task.dart';
+import 'package:cocoon_service/src/model/firestore/task.dart' as f;
```
Comment (ricardoamador) [inline app_dart/test/request_handlers/update_task_status_test.dart:7]: Same comment as above for the import name.

**Review by ricardoamador — APPROVED** [review]
LGTM with nits.

File: app_dart/test/request_handlers/update_task_status_test.dart (hunk 2 above)
Comment (keyonghan) [reply app_dart/test/request_handlers/update_task_status_test.dart]: Done

**Pull request merged**